src/dmd/dsymbol.d

```diff
@@ -595,7 +595,7 @@ extern (C++) class Dsymbol : ASTNode
         return followInstantiationContext(p1, p2) ? toParent2() : toParentLocal();
     }
 
-    final inout(TemplateInstance) isInstantiated() inout
+    final inout(TemplateInstance) isInstantiated() inout pure nothrow
```

Comment:
I would advise against trying to add attributes to DMD, as the next PR might need to remove them.
But this seems completely unrelated with the PR at hand.

Reply:
Removed.